CONTRIBUTING.md

```diff
@@ -1,35 +1,26 @@
 # Web Platform Working Group
 
-Contributions to this repository are intended to become part of
-the W3C's HTML Recommendation, and are governed by the
-	[W3C Patent Policy](http://www.w3.org/Consortium/Patent-Policy-20040205/)
-and
-	[Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
-  
-Editorial improvements are welcome, just make a pull request! If you are not a member of the
-W3C [Web Platform Working Group](https://www.w3.org/WebPlatform/WG/) you may see a warning about IPR commitments, but this will be
-handled by the editorial team.
-
-Some issues are marked as [good first issues](https://github.com/w3c/html/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-or [help wanted](https://github.com/w3c/html/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
-If you want to help but are unsure how to begin, these might be good places to start. 
-
-You should read the documentation on 
-[editing conventions and the source documents](https://github.com/w3c/html/blob/master/docs/contributing-documentation.md), 
-and you may want to read [how the spec is built](https://github.com/w3c/html/blob/master/docs/build-documentation.md)
-  
-To make **substantive contributions** that clarify or change conformance requirements, you must either participate
-in the W3C [Web Platform Working Group](https://www.w3.org/WebPlatform/WG/)
-or make a	[non-member patent licensing commitment](https://www.w3.org/2004/01/pp-impl/83482/nmlc).
-
-If your contribution includes the work of others, please see https://github.com/w3c/html/blob/master/docs/contributing-documentation.md
-for further guidance.
-
-# Tests
-
-For normative changes, a corresponding
-[web-platform-tests](https://github.com/w3c/web-platform-tests) PR is highly appreciated. Typically,
-both PRs will be merged at the same time. Note that a test change that contradicts the spec should
-not be merged before the corresponding spec change. If testing is not practical, please explain why
-and if appropriate [file an issue](https://github.com/w3c/web-platform-tests/issues/new) to follow
-up later. Add the `type:untestable` or `type:missing-coverage` label as appropriate.
+Contributions to this repository are intended to become part of the World Wide Web Consortium's (W3C) [HTML Recommendation](https://www.w3.org/TR/html52/), and are governed by the [W3C Patent Policy](http://www.w3.org/Consortium/Patent-Policy-20040205/) and [Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
```

Comment:
Hmm. maybe instead of HTML 5.2 we should link to the latest recommendation: https://www.w3.org/tr/html so it updates...

Reply:
Ah, good call.